### ISSUE TITLE
Troubles finding loss function definition and training loop function

### ISSUE THREAD

**Issue by dunzeng** [DeepLearnXMU/LLaVE#12]
Hello, thank you for your great work!

I'm interested in the implementation details of rewards. I didn't find the loss function definition and training loop while I tracked the function calls started from llava/train/train_mem.py to llava/train/train.py.

Could you kindly point out them for me?

**Comment by dunzeng** [comment]
found at https://github.com/XMUDeepLIT/LLaVE/blob/main/llava/model/language_model/loss.py
